src/dataset-builder/DatasetBuilder.ts

```diff
@@ -726,16 +732,18 @@ export const DatasetBuilderView: React.FC<DatasetBuilderProps> = (props) => {
                   onStateChange,
                   updateCohorts: setCohorts,
                   updateConceptSets: setConceptSets,
-                  dataset: datasetModel.state,
+                  snapshotId,
+                  snapshotBuilderSettings: snapshotBuilderSettings.state,
                   cohorts,
                   conceptSets,
                 });
               case 'cohort-editor':
-                return datasetModel.state.snapshotBuilderSettings
+                return snapshotBuilderSettings.status === 'Ready'
```

Comment:
I don't think you need this check, since it is tested on line 723 before render

Reply:
Good point!